modules/solid_mechanics/doc/content/source/timeintegrators/ExplicitMixedOrder.md

```diff
@@ -55,6 +61,12 @@ An example input file is shown below:
         matrix_tags = 'system'
         variable = disp_x
     []
+    [dampingmatrix]
+        type = MassMatrix
+        density = 0.8
+        matrix_tags = 'damping'
+        variable = disp_x
+    []
```

Comment:
@maxnezdyur @TheGreatCid 

Do you think we should rename `MassMatrix` to be something more general, or just create another object called `DampingMatrix`?

Reply:
Sorry, I'm late to this, but I agree with Max. Makes sense to leave as is.